File: Selenium/Project/Java/Activity5.java
package project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

import java.util.List;

public class Activity5 {
    WebDriver driver;
    @BeforeClass
    public void setUp()
    {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs/");
    }
    @Test
    public void verifyJobsTitle()
    {
        List<WebElement> listOfNavigation = driver.findElements(By
                .xpath("//ul[contains(@id,'primary-menu')]//li//a"));
        for(WebElement we : listOfNavigation)
        {
            if(we.getText().equals("Jobs"))
            {
                we.click();
                break;
            }
        }
        String jobsTitle = driver.getTitle();
        Assert.assertEquals(jobsTitle,"Jobs – Alchemy Jobs");
    }
    @AfterClass
    public void closeBrowser()
    {
        driver.close();
    }
}

File: Selenium/Project/Python/Activity5.py
import pytest
from selenium.webdriver.common.by import By

@pytest.mark.usefixtures("setup")
class TestSite:

    def test_verify_jobs_title(self):
        self.driver.get("https://alchemy.hguy.co/jobs/")
        listNavigation = self.driver.find_elements(By.XPATH,
                                          "//ul[contains(@id,'primary-menu')]//li//a")
        for we in listNavigation:
            if we.text == "Jobs":
                we.click()
                break
        jobsTitle = self.driver.title
        print(jobsTitle)
        assert jobsTitle == "Jobs – Alchemy Jobs"



